snippets/router.cson

```diff
@@ -0,0 +1,17 @@
+'.source.coffee':
+  'Nested resource':
+    'prefix': '@resource-v'
```

Comment:
it is for cases they are tried to type what they often do.
It set a trap ;)
they start to type @reso... they are offered with snippet above.